libnavigation-router/src/main/java/com/mapbox/navigation/route/internal/util/DirectionsRouteRefreshUtils.kt

```diff
@@ -0,0 +1,21 @@
+package com.mapbox.navigation.route.internal.util
+
+import com.mapbox.api.directionsrefresh.v1.models.DirectionsRefreshResponse
+import com.mapbox.api.directionsrefresh.v1.models.DirectionsRouteRefresh
+import com.mapbox.bindgen.Expected
+import com.mapbox.bindgen.ExpectedFactory
+
+fun parseDirectionsRouteRefresh(
```

Comment:
```suggestion
internal fun parseDirectionsRouteRefresh(
```
and move to a regular package. Unless any other module would need to use this utility?

Reply:
This module does not have a regular package, so I didn't move the file.